ci-operator/step-registry/ipi/upgrade/aws/ipi-upgrade-aws-workflow.yaml

```diff
@@ -0,0 +1,20 @@
+workflow:
+  as: ipi-aws
```

Comment:
Should this be `ipi-upgrade-aws` to match the directory?  Also, I'm fine leaving this workflow out until we have a consumer for it.